src/main/resources/fabric.mod.json

```diff
@@ -1,4 +1,14 @@
 {
+	"schemaVersion": 1,
+	"id": "zoom-o-matic",
+	"version": "${version}",
+	"name": "Zoom-o-Matic",
+	"description": "A simple, but customizable multi-button zoom tool for Minecraft.",
+	"contact": {
+		"sources": "https://github.com/NicoIsLost/Zoom-o-Matic",
+		"issues": "https://github.com/NicoIsLost/Zoom-o-Matic/issues",
+		"discord": "https://discord.gg/66rV5hanFP"
+	},
 	"custom": {
```

Comment:
Fabric was complaining that `schemeVersion` wasn't the first element, so I moved it.  No idea if that changes anything, but that's why